interceptor_ext_test.go

```diff
@@ -139,6 +139,33 @@ func TestOnionOrderingEndToEnd(t *testing.T) {
 	assert.Nil(t, responses.Close())
 }
 
+func TestEmptyUnaryInterceptorFunc(t *testing.T) {
+	t.Parallel()
+	mux := http.NewServeMux()
+	interceptor := connect.UnaryInterceptorFunc(func(next connect.UnaryFunc) connect.UnaryFunc {
+		return func(ctx context.Context, request connect.AnyRequest) (connect.AnyResponse, error) {
+			return next(ctx, request)
+		}
+	})
+	mux.Handle(pingv1connect.NewPingServiceHandler(pingServer{}, connect.WithInterceptors(interceptor)))
+	server := httptest.NewServer(mux)
+	t.Cleanup(server.Close)
+	connectClient := pingv1connect.NewPingServiceClient(server.Client(), server.URL, connect.WithInterceptors(interceptor))
+	_, err := connectClient.Ping(context.Background(), connect.NewRequest(&pingv1.PingRequest{}))
+	assert.Nil(t, err)
+	sumStream := connectClient.Sum(context.Background())
+	assert.Nil(t, sumStream.Send(&pingv1.SumRequest{Number: 1}))
+	resp, err := sumStream.CloseAndReceive()
+	assert.Nil(t, err)
+	assert.NotNil(t, resp)
+	countUpStream, err := connectClient.CountUp(context.Background(), connect.NewRequest(&pingv1.CountUpRequest{}))
+	assert.Nil(t, err)
+	for countUpStream.Receive() {
+		assert.NotNil(t, countUpStream.Msg())
+	}
+	defer countUpStream.Close()
```

Comment:
We're already at the end of the function, so the defer isn't doing anything. Should this just be `assert.Nil(countUpStream.Close()`?